joomla/joomla-cms
# [4.4] Use .env files for the node tools

Pull Request for Issue # .

### Summary of Changes

- Adding a `production.env` and a `development.env` file for the build env variables
- Adjust the script tags on the package.json

### Testing Instructions

- add a `console.log(process.env.NODE_ENV)` after https://github.com/joomla/joomla-cms/blob/12ce681c526a3517c165ba1003246f9c01674808/build/build-modules-js/javascript/build-com_media-js.es6.js#L14
- Run the commands `npm run build:com_media` should see `PRODUCTION`
- Run the commands `npm run build:com_media:dev` should see `DEVELOPMENT`
- Run the commands `npm run update` should see `PRODUCTION`
- All commands produce the file `media/com_media/js/media-manager.min.js`

### Actual result BEFORE applying this Pull Request
???


### Expected result AFTER applying this Pull Request
All good


### Link to documentations
Please select:
